examples/scope_example.cpp

```diff
@@ -1,10 +1,13 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+#include <beman/scope/scope.hpp>
+
 #include <cstdlib>
-#include <experimental/scope>
 #include <iostream>
 #include <string_view>
 
+namespace scope = beman::scope;
```

Comment:
If you want, we can update the alias here as
```suggestion
namespace scope = std::experimental;
```

@JeffGarland

Reply:
I think we should just do the using on the 4 classes in the experimental.  Also, we need the includes -- scope and memory resource.

Reply:
alright. I will get working on that.